src/Unity.Mathematics/Noise/cellular2x2.cs

```diff
@@ -10,12 +10,12 @@ namespace Unity.Mathematics
 {
     public static partial class noise
     {
-        // Cellular noise, returning F1 and F2 in a float2.
-        // Speeded up by umath.sing 2x2 search window instead of 3x3,
-        // at the expense of some strong pattern artifacts.
-        // F2 is often wrong and has sharp discontinuities.
-        // If you need a smooth F2, use the slower 3x3 version.
-        // F1 is sometimes wrong, too, but OK for most purposes.
+        /// <summary>
+        /// 2D Cellular noise ("Worley noise") with a 2x2 search window. Faster than using 3x3, at the expense of some strong pattern artifacts.
+        /// F2 is often wrong and has sharp discontinuities. If you need a smooth F2, use the slower 3x3 version. F1 is sometimes wrong, too, but OK for most purposes.
+        /// </summary>
```

Comment:
`<summary>` tags are supposed to be one liners so I suggest this summary section to be just: 2D Cellular noise ("Worley noise") with a 2x2 search window.

And then the rest can be in `<remarks>`